turtlebot3_sim_tests/resources/turtlebot_nav.rossystem

```diff
@@ -0,0 +1,617 @@
+RosSystem { Name 'demo'
+    RosComponents ( 
+
+        ComponentInterface { name '/gazebo'
+            RosParameters {
+                RosParameter '/gazebo/sor_pgs_precon_iters' {RefParameter './gazebo./gazebo./gazebo/sor_pgs_precon_iters' value 0},
```

Comment:
@ipa-hsd I tried rosgraph_monitor to create rossystem. When I load it in rostooling. I got some errors, for example here. 

Whast is the meaning of "RefParameter './gazebo./gazebo./gazebo/sor_pgs_precon_iters'? Is it should be 'gazebo.gazebo.gazebo/sor_pgs_precon_iters'?